docs/cloud/what-are-the-default-limits-for-temporal-cloud.md

```diff
@@ -0,0 +1,42 @@
+---
+id: what-are-the-default-limits-for-temporal-cloud
+title: Default limits for Temporal Cloud
+sidebar_label: Cloud limits
+description: The default experience in Temporal Cloud has limits, some of which are configurable.
+---
+
+<!--- What are the default limits for Temporal Cloud? --->
+
+The default experience in Temporal Cloud has the following limits.
+Some of these limits are configurable.
+
+<!--- What are the default limits for accounts in Temporal Cloud? --->
+
+## Account default limits
+
+|                                      | Default                | Maximum (self-serve) | Maximum (system)                     |
+| ------------------------------------ | ---------------------- | -------------------- | ------------------------------------ |
+| Audit Log Retention Period           | 7 days                 |                      | 7 days                               |
```

Comment:
i don't know if we want to communicate this - we start sending the logs right away, so my feeling is we would want to remove this one for the time being, and let's chat internally what we want to publish here.